products/models.py

```diff
@@ -0,0 +1,64 @@
+from django.db import models
+
+class Category(models.Model):
+    name = models.CharField(max_length=45)
+
+    class Meta:
+        db_table = 'categories'
+    def __str__(self):
+        return self.name
+
```

Comment:
Meta와 def __str__: 사이에 간격 추가해 주세요. 
이 부분 말고도 다른 부분들도 다 한 번씩 봐주세요.
```suggestion
    class Meta:
        db_table = 'categories'
        
    def __str__(self):
        return self.name

```